sbt_common/internal_model/src/test/scala/uk/ac/wellcome/models/work/internal/WorkTest.scala

```diff
@@ -0,0 +1,130 @@
+        |  "description": null,
+        |  "physicalDescription": null,
+        |  "extent": null,
+        |  "lettering": null,
+        |  "createdDate": null,
+        |  "subjects": [],
+        |  "genres": [],
+        |  "contributors": [
+        |    {
+        |      "agent": {
+        |        "agent": {
+        |          "label": "Science Museum, London",
+        |          "type": "Agent"
+        |        },
+        |        "type": "Unidentifiable"
+        |      },
+        |      "roles": [],
+        |      "ontologyType": "Contributor"
+        |    }
+        |  ],
+        |  "thumbnail": {
+        |    "url": "https://iiif.wellcomecollection.org/image/L0057464.jpg/full/300,/0/default.jpg",
+        |    "license": {
+        |      "id": "cc-by",
+        |      "label": "Attribution 4.0 International (CC BY 4.0)",
+        |      "url": "http://creativecommons.org/licenses/by/4.0/",
+        |      "ontologyType": "License"
+        |    },
+        |    "locationType": {
+        |      "id": "thumbnail-image",
+        |      "label": "Thumbnail Image",
+        |      "ontologyType": "LocationType"
+        |    },
+        |    "credit": null,
+        |    "ontologyType": "DigitalLocation",
+        |    "type": "DigitalLocation"
+        |  },
+        |  "production": [],
+        |  "language": null,
+        |  "dimensions": null,
+        |  "items": [
+        |    {
+        |      "canonicalId": "hy2cbxkh",
+        |      "sourceIdentifier": {
+        |        "identifierType": {
+        |          "id": "miro-image-number",
+        |          "label": "Miro image number",
+        |          "ontologyType": "IdentifierType"
+        |        },
+        |        "ontologyType": "Item",
+        |        "value": "L0057464"
+        |      },
+        |      "otherIdentifiers": [],
+        |      "locations": [
+        |        {
+        |          "url": "https://iiif.wellcomecollection.org/image/L0057464.jpg/info.json",
+        |          "license": {
+        |            "id": "cc-by",
+        |            "label": "Attribution 4.0 International (CC BY 4.0)",
+        |            "url": "http://creativecommons.org/licenses/by/4.0/",
+        |            "ontologyType": "License"
+        |          },
+        |          "locationType": {
+        |            "id": "iiif-image",
+        |            "label": "IIIF image",
+        |            "ontologyType": "LocationType"
+        |          },
+        |          "credit": "Science Museum, London",
+        |          "ontologyType": "DigitalLocation",
+        |          "type": "DigitalLocation"
+        |        }
+        |      ],
+        |      "ontologyType": "Item"
+        |    }
+        |  ],
+        |  "version": 27,
+        |  "visible": true,
+        |  "ontologyType": "Work"
+        |}
+      """.stripMargin
+
+    intercept[GracefulFailureException] {
+      fromJson[IdentifiedWork](jsonString).get
+    }
```

Comment:
Is it possible to be clearer what the failure is?

Reply:
I need to rebase, I’ll include a check on the error message